### ISSUE TITLE
Access for Josh Kim (new macbook)

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#29889]
# Instructions
1. Fill out **Description**
2. Change the **Title** to include your name
3. Add the label used by your team (eg: `BAH-526`)
4. Do not remove `operations` label or the `ops-access-request` label; this will notify the VSP Operations team of your request
5. When the issue is closed you will be notified and can continue with on-boarding setup

# Description
- Please provide your Contracting Officer's Representative (COR) name.
   > Ariel Nerbovig 
- Please provide your Vendor Onboarding Representative name. This is the person from the contracting company responsible for your onboarding process.
   > Ariel Nerbovig 
- Please provide your name.
   > Josh Kim
- Please Provide your email.
   > josh.kim@adhocteam.us
- Please provide the name of your team, your role on that team, and the name of the company you work for.
   > VSF a11y specialist, Ad Hoc

- Requesting SSH key authorization?
   > **YES** 
   - What is your **public** SSH key? If you don't have a SSH Key or know how to find it [click here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#create-ssh-public-key) for instructions. (Example below)

```
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC/5er4BXlOq6PYaeULy69SmTU4LSlBe2q4FBbw09dRDJFlqXN1bY/Q7oEF+t2sHe/HzIUar1njdVLtgjT0RpSgcxNGUzSwTAOElgxyYvwl+yLKv1uzL/XIEbiiOd6ePDoEg+bnTnoyekUuEs4/q6Q1fydMZNQuqtPJx/DjO2dw0vyGCMDzk+CjzXeyy5HMjyW3EELQIQJrZ1Bm+sdxbl3fsHi//nZBOsiq7UBIRfviubRZ0ytoZdKHfJPjVxLaxTfRW3D9YnLidkWHiVfdsGwN4W3wkggcOpEh23MbkpkcmGxef8hGodeTDmi/+7ApaxsfRf36DCm7xg0WmhqS2H2xi+unphGd5xZPXNv3ENHDWNXfbao7H4yHQeW0OLJI7nPq1BVmq+Eq7/DLjVY4vimGayXtkxvGLjuqlykrKGNiF+IsCAswrsngHLF5i0rkdynkOYl3u/69aeg07tJ4xZp0W+NMTKfQkyxBM/mFjorBU7ytJwoxThW96Kt5Is+E7Qk= joshkim@MacBook-Pro
```

- Grant AWS Console access?
  > NO
  - What AWS services do you need access to and why?

**Comment by jbritt1** [comment]
@joshkimux, is this key to be added in addition to the existing key we have on file for you or is it a total replacement?

**Comment by joshkimux** [comment]
@jbritt1 A total replacement, my old macbook bricked 🙇

**Comment by jbritt1** [comment]
> @jbritt1 A total replacement, my old macbook bricked 🙇

Sorry to hear that, we will get that key updated for you.

**Comment by jbritt1** [comment]
Key updated @joshkimux, give it about 10 minutes before attempting a new connection. Feel free to reach out over Slack in #vfs-platform-support or reopen this ticket if any issues are observed.

**Comment by joshkimux** [comment]
It's working great! Thank you again!

**Comment by jbritt1** [comment]
> It's working great! Thank you again!

Anytime!
